src/surface.rs

```diff
@@ -24,27 +25,17 @@ impl<T: 'static> fmt::Debug for Surface<T> {
 pub enum FrontBufferError {
     /// An unknown error happened
     Unknown,
```

Comment:
If this is the only remaining variant, maybe let's do a `pub struct FrontBufferError;` instead.